packages/extension-chakra-storefront/src/theme/components/base/heading.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {defineRecipe} from '@chakra-ui/react'
+
+export default defineRecipe({
+    base: {
+        fontWeight: 'bold'
```

Comment:
Chakra v3 has changed default fontWeight from bold to semobold for all heading. We modified it here to match current design on pwa kit v3